buildSrc/src/main/java/com/osacky/flank/gradle/YamlConfigWriterTask.kt

```diff
@@ -3,13 +3,20 @@ package com.osacky.flank.gradle
 import org.gradle.api.DefaultTask
 import org.gradle.api.file.ProjectLayout
 import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.OutputFile
 import org.gradle.api.tasks.TaskAction
+import java.io.File
 import javax.inject.Inject
 
-open class YamlConfigWriterTask @Inject constructor(private val config: FladleConfig, private val extension: FlankGradleExtension, private val projectLayout: ProjectLayout) : DefaultTask() {
+open class YamlConfigWriterTask @Inject constructor(private val config: FladleConfig, private val extension: FlankGradleExtension, projectLayout: ProjectLayout) : DefaultTask() {
 
   private val yamlWriter = YamlWriter()
 
+  private val fladleDir = projectLayout.fladleDir.get().asFile
+
+  @OutputFile
+  val fladleConfigFile: File = fladleDir.resolve("flank.yml")
```

Comment:
is it possible to make this simply a property so that it can be lazily evaluated?

Reply:
Isn't creating a File almost free? It's only when you try to access it that it involves the IO.
I can make it lazy but I don't see the point (maybe I'm missing something)

Reply:
"almost" :D